docs/VAULT.md

```diff
@@ -0,0 +1,30 @@
+# Using Vault to keep tabs on environment variables
+
+## Prerequisites
+
+- An instance of [Hashcorps Vault](https://www.vaultproject.io/) running
```

Comment:
`Hashicorp`'s maybe =) (Lacking the 'i').

Reply:
fixed